src/main/java/com/google/javascript/Main.java

```diff
@@ -0,0 +1,24 @@
+package com.google.javascript;
```

Comment:
com.google.javascript.Main is a bit of an odd package/class name. Maybe move it into a separate subpackage?

Reply:
I'm going to try to see if I can make separate Jars, one for gents and one for clutz, so the Main class isn't needed.

Reply:
It is trickier that I thought it would be to have gradle build two jars for the same project.  So I think this approach is the right one to take now.  I don't want to spend more time on this.  I'm fine with changing the name from `com.google.javascript.Main`.  What do you think would be a better name?

Reply:
How about `LinkedJarEntry` or something of sorts. Even more import than the name, add a comment describing that you are adding this for testing/debugging convenience and it is not the intended entry point to either clutz or gents.

Reply:
Sounds good.  I'll also add docs.